content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -62,7 +62,7 @@ with great caution. The following describes how to quickly experiment with Valid
 The following is an example of a ValidatingAdmissionPolicy.
 
 ```yaml
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?